docs/development.md

```diff
@@ -7,9 +7,9 @@
 - [npm](https://nodejs.org/en/download/)
 
 ```bash
-git clone git@github.com:drupal-http-apis/contenta_jsonapi.git
-cd contenta_jsonapi
-sh scripts/development/build-contenta_jsonapi.sh
+$ git clone git@github.com:drupal-http-apis/contenta_jsonapi.git
```

Comment:
good point, I was thinking more on readability, yet it prefer not to impact the ability to copy and paste, removing it.